### DISCUSSION TITLE
Ghost folders after syncing

### DISCUSSION THREAD

**Discussion by Section1977** [minio/minio#18617] [Q&A] [answered]
Minio works quite fine.
But if I use Nextcloud Client using external S3 storage there is something strange happening:
If I create a folder with Nextcloud Client, then this folder is being created in S3 storage. So far so good.
If I connect with Minio Dashboard to the S3 storage and click on the created folder, I get into the folder but the folders name is shown again and clickable but with no function.
The content of the folder stays visible in this context.

This is different and weird to normal behaviour of S3 storage.
Please help or fix.

**Comment by harshavardhana** [comment, marked as answer]
How are we supposed to know how you are running MinIO? And what is the MinIO version? Are we supposed to imagine your setup?

We have an issue template for a reason so that you can provide us proper details.
